content/SCALE/SCALEUIReference/Apps/InstallCustomAppScreens.md

```diff
@@ -0,0 +1,274 @@
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppScreenStorage.png" alt="Storage Settings" id="11: Storage Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppAddHostPathVol.png" alt="Host Path Volume Settings" id="12: Host Path Volume Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Host Path Volumes** | Use to configure a persistent host path volume. Click **Add** to display a block of **Host Path Volume** settings. |
+| **Host Path** | Required. Enter or click <span class="material-icons">arrow_right</span> to the left of <span class="material-icons">folder</span> **/mnt** to browse to the location of the dataset to populate the **Mount Path**. Click on the dataset to select and display it in the **Mount Path** field. |
+| **Mount Path** | Required. Enter the <file>**/data**</file> directory where the host path mounts inside the pod. |
+| **Read Only** | Select to make the mount path inside the pod read-only and prevent using the container to store data. Kubernetes recommends scoping it to only require file or directory and mounting it as read only. |
+{{< /truetable >}}
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppAddMemoryBackedVol.png" alt="Memory Backed Volume Settings" id="13: Memory Backed Volume Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Memory Backed Volumes** | Use to configure a Kubernetes emptydir volume that mounts inside the pod. Click **Add** to display a block of **Memory Backed Volume** settings. |
+| **Mount Path** | Required. Enter the path where the Kubernetes emptydir volume mounts inside the pod. |
+| **Size Limit** | Optional. Enter the size of the volume. Use the format, *100Mi, 1Gi, 2Gi*, etc. |
+{{< /truetable >}}
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppAddStorageVolumes.png" alt="Volume Settings" id="14: Volume Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Volumes** | Use to configure a persistent volume that mounts inside the pod. Click **Add** to display a block of **Volumes** settings. Click again to add another block of settings. |
+| **Mount Path** | Required. Enter the path where the persistent volume mounts inside the pod. |
+| **Dataset Name** | Required. Enter the name of the dataset created for the storage volume. |
+{{< /truetable >}}
+
+{{< /expand >}}
+
+### Workload Details Settings
+**Workload Details** settings specify how to deploy workloads in the container (pod). 
+Kubernetes defines workloads as applications running in the pod.
+**Workload Details** settings specify if containers in a pod run with TTY or STDIN enabled, allow enabling any device on the host, or configuring host capabilities, and if you run the container as a user or group.
```

Comment:
This list is a little confusing with multiple conjuntions. I think removing one of the commas fixes it, but I'm not sure if that's what you were going for.

```suggestion
**Workload Details** settings specify if containers in a pod run with TTY or STDIN enabled, allow enabling any device on the host or configuring host capabilities, and if you run the container as a user or group.
```